src/services/legal-services.ts

```diff
@@ -133,6 +133,30 @@ export default class LegalServices {
       })
   }
 
+  /**
+   * Fetches name request data with phone and email validation.
+   * @param nrNumber the name request number (eg, NR 1234567)
+   * @param phone the name request phone (eg, 12321232)
+   * @param email the name request email (eg, ab@gmail.com)
```

Comment:
This is probably someone's actual email address, hahaha.

Use `@example.com` as a non-routable domain for test email addresses.

Reply:
Fixed, thanks.